spring-kafka/src/main/java/org/springframework/kafka/support/converter/MessagingMessageConverter.java

```diff
@@ -37,7 +39,7 @@
  * @author Marius Bogoevici
  * @author Gary Russell
  */
-public class MessagingMessageConverter<K, V> implements MessageConverter<K, V> {
+public class MessagingMessageConverter<K, V> implements MessageConverter {
```

Comment:
My IDEA says me that the generics are redundant.
Investigating why you need them at all...